objects/oplayercontroller/Step_0.gml

```diff
@@ -64,14 +64,39 @@ hurtbox.image_yscale = 32;
 hurtboxOffset = -8;
 
 
+// Handle storing input for Super Jump
+if (superJumpTimer > 0) 
+{
+	superJumpTimer--;
+} 
+else
+{
+	storedSuperJump = false;
+}
+if (verticalMoveDir == -1)

```

Comment:
Please put a linebreak here to help visually separate one if...else from the other.